gulpfile.js

```diff
@@ -0,0 +1,128 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview node module that automate tasks.
+ *
+ * @author barnabasmakonda@gmail.com (Barnabas Makonda)
+ */
+
+var argv = require('yargs')
+    .usage(
+      'Usage: $0 --gae_devserver_path [path to app engine]' +
```

Comment:
Add spaces at the end of these lines. The usage message comes out wrong (as you can see if you run python scripts/build.py).